Dockerfile_Fedora

```diff
@@ -3,16 +3,15 @@ FROM fedora:latest
 
 
 LABEL project="gluster-build"
-LABEL maintainer "sajmoham@redhat.com"
+LABEL maintainer "gluster-users@gluster.org"
 
-RUN dnf -y update && dnf clean all
+RUN dnf -y --setopt tsflags=nodocs install automake autoconf libtool flex bison openssl-devel  \
+    libxml2-devel python-devel libaio-devel libibverbs-devel librdmacm-devel readline-devel \
```

Comment:
libibverbs-devel , ibcmocka-devel - are not needed.

Reply:
oh my bad !!!!